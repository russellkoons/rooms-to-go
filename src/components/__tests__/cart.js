import React from "react"
import renderer from "react-test-renderer"

import Cart from "../../pages/cart"

describe("Cart", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Cart cart={[]} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})