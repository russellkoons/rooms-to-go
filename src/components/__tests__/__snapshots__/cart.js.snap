// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart renders correctly 1`] = `
Array [
  <h1>
    Cart
  </h1>,
  <p>
    There's nothing here...
  </p>,
]
`;
